cmd/repo_remove_test.go

```diff
@@ -44,8 +44,10 @@ func TestRepoRemoveLogs(t *testing.T) {
 			sandbox, cleanup := cmdtest.TestSetupWithSandbox(t, true)
 			defer cleanup()
 
+			sandbox.SetConfigInTestData(tt.configDir)
+
 			// see how many repos we currently have
-			startRepos := getRepoListOutput(t, sandbox)
```

Comment:
again, why delete this and the endRepos check?

Reply:
same comment as before, I think these are important and we need to get it working with them in there